DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
I know, before sending this PR I discussed this in a BTV software meeting with @imarches, @pvmulder, and @acaudron, and outlined the problem in [this](https://indico.cern.ch/event/441479/session/0/contribution/8/attachments/1145703/1642478/BTV_CTagDev_27Aug2015.pdf) presentation (slide 9).
Also, if you look in [TaggingVariable.h](https://github.com/mverzett/cmssw/blob/ctagging_76Integration/DataFormats/BTauReco/interface/TaggingVariable.h) you will notice that "trackIp2dVal" never existed, and that name was a bug.

I checked by running the full BTagging sequence that the commit was non-breaking before pushing it in anyways. 

For the future, always in that presentation, I suggested to move the TaggingVariableToken to be a (unordered_)map to make this sort of mistake more difficult to make. What do you think about it?